src/testFixtures/groovy/grails/plugin/geb/RecordingSettings.groovy

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright 2024 original author or authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package grails.plugin.geb
+
+import groovy.transform.CompileStatic
+import groovy.transform.Memoized
+import groovy.util.logging.Slf4j
+import org.testcontainers.containers.BrowserWebDriverContainer
+import org.testcontainers.containers.VncRecordingContainer
+
+import static org.testcontainers.containers.BrowserWebDriverContainer.*
+import static org.testcontainers.containers.VncRecordingContainer.*
```

Comment:
I don't think you need to star-import. You are not actually using the values directly.
It's enough to statically import `VncRecordingMode` and `VncRecordingFormat`
```groovy
import static org.testcontainers.containers.BrowserWebDriverContainer.VncRecordingMode
import static org.testcontainers.containers.VncRecordingContainer.VncRecordingFormat
```

Reply:
I just assumed IntelliJ did this by default, not sure why it did `*`.  Thanks for catching this.